test/module/irohad/ametsuchi/postgres_block_storage_test.cpp

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "ametsuchi/impl/postgres_block_storage.hpp"
+#include "ametsuchi/impl/postgres_block_storage_factory.hpp"
+
+#include "backend/protobuf/proto_transport_factory.hpp"
+#include "module/irohad/ametsuchi/ametsuchi_fixture.hpp"
+#include "module/shared_model/builders/protobuf/test_block_builder.hpp"
+#include "module/shared_model/builders/protobuf/test_transaction_builder.hpp"
+#include "module/shared_model/interface_mocks.hpp"
+#include "module/shared_model/validators/validators.hpp"
+
+using namespace iroha::ametsuchi;
+using namespace shared_model::validation;
+
+using ::testing::NiceMock;
+using ::testing::Return;
+using ::testing::ReturnRef;
+
+using MockBlockIValidator = MockValidator<shared_model::interface::Block>;
+using MockBlockPValidator = MockValidator<iroha::protocol::Block_v1>;
+
+class PostgresBlockStorageTest : public AmetsuchiTest {
```

Comment:
Consider to not use Ametsuchi test.

Reply:
I think we can leave this test as it is, till the new shiny storage itf is delivered.

Reply:
...just to save the time on reinventing test classes. all similar tests are going to be rewritten with the new itf in bulk anyway.